playground/frontend/playground_components/lib/src/widgets/run_button.dart

```diff
@@ -31,67 +32,100 @@ const _width = 150.0;
 
 class RunButton extends StatelessWidget {
   final PlaygroundController playgroundController;
-  final bool isRunning;
   final VoidCallback runCode;
   final VoidCallback cancelRun;
   final bool disabled;
```

Comment:
Consider flipping to `isEnabled`,
https://dart.dev/guides/language/effective-dart/design#prefer-the-positive-name-for-a-boolean-property-or-variable